Common/GPU/Vulkan/VulkanDescSet.cpp

```diff
@@ -68,14 +69,17 @@ VkDescriptorSet VulkanDescSetPool::Allocate(int n, const VkDescriptorSetLayout *
 		result = vkAllocateDescriptorSets(vulkan_->GetDevice(), &descAlloc, &desc);
 		_assert_msg_(result == VK_SUCCESS, "Ran out of descriptor space (frag?) and failed to allocate after recreating a descriptor pool. res=%d", (int)result);
 	}
+	usage_++;
```

Comment:
There's another one just below.  Intentional to have two?  Seems probably better in the other place.

-[Unknown]

Reply:
Oh, this was a merge between two branches where I did the same fix in both..